### PR TITLE
[opt](optimizer) Recycle expired table stats

### PR DESCRIPTION
**Pull request by Kikyou1997**
## Proposed changes

Remove table stats when olap table is dropped

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by Kikyou1997** [comment]
run buildall

**Comment by Kikyou1997** [comment]
run buildall

**Comment by Kikyou1997** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/journal/JournalEntity.java [hunk 1]
```diff
@@ -896,6 +897,11 @@ public void readFields(DataInput in) throws IOException {
                 isRead = true;
                 break;
             }
+            case OperationType.OP_DELETE_TABLE_STATS: {
```
Comment (morrySnow) [inline fe/fe-core/src/main/java/org/apache/doris/journal/JournalEntity.java:900]: do we need add this operationtype to doris-2.0 branch?

**Review by morrySnow — APPROVED** [review]

**Review by englefly — APPROVED** [review]

File: fe/fe-core/src/main/java/org/apache/doris/journal/JournalEntity.java (hunk 1 above)
Comment (Kikyou1997) [reply fe/fe-core/src/main/java/org/apache/doris/journal/JournalEntity.java:900]: I think we need

**Pull request merged**